mysql-test/lib/My/Debugger.pm

```diff
@@ -144,7 +144,10 @@ sub do_args($$$$$) {
   my $v = $debuggers{$k};
 
   # on windows mtr args are quoted (for system), otherwise not (for exec)
-  sub quote($) { $_[0] =~ /[; ]/ ? "\"$_[0]\"" : $_[0] }
+  sub quote($) {
+    my $tmp= $_[0];
+    $tmp =~ s/^"(.*)"$/$1/;
```

Comment:
this is wrong. if the parameter itself includes double quotes, they have to be preserved, not stripped.

Let's do a simple fix and only add `>` to the line 147 ?

Reply:
Ok, have updated.